02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>this bit about slicing hierarchies is confusing for me, as I'm used to seeing them used for dates, and not for other things.</p><p>Check with Hetal and see what she suggests doing.</p><p>There may be real use cases when you would use a slicing hierarchy for something that is not a date, but I can't see why or when </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>Dates is a more canonical topic to use for explaining slicing hierarchies for sure.</p><p> </p><p>We don't need to lose this bit, but it might be worth separating the two sections out: slicing hierarchies using a simple example, then <code>create_parameter_hierarchy_from_column</code> , then mention it creates a slicing hierarchy by default "for free"</p>

Reply:
<p>The <code>Using slicing hierarchies</code> section has been re-designed to include a more basic example with <code>AsOfDate</code> and inventory amounts. The content for <code>create_parameter_hierarchy_from_column</code> has been taken out and will most likely be moved to the 3rd part notebook for hierarchies, which will refer to parameter hierarchies and source simulations.</p>